packages/angular-cli/models/webpack-build-utils.ts

```diff
@@ -109,3 +109,18 @@ export function extraEntryParser(
       return extraEntry;
     });
 }
+
+const hashFormats: { [option: string]: any } = {
+  'none':     { chunk: '',             extract: '',                  file: '[name]'    },
+  'media':    { chunk: '',             extract: '',                  file: '[hash:20]' },
```

Comment:
> How about something in the middle: 12?

Sounds good to me if you dont mind also editing the test cases.

> I agree on the name inclusion as it would be consistent with the others but was hesitant to change the existing paradigm without a consensus.

Lets get a green light from one of the maintainers. File- and url-loader should emit a filename containing `[name].[hash:12]` or `[name].[hash:20]` for consistency and human readability.